components/Tour/StageMapLine.tsx

```diff
@@ -0,0 +1,81 @@
+import { Location } from "@/model/model";
+import MapboxGL from "@rnmapbox/maps";
+import { lineString, point, featureCollection } from "@turf/helpers";
+import { Layout, useTheme } from "@ui-kitten/components";
+import React from "react";
+import { Feature, FeatureCollection, LineString, Point } from "geojson";
```

Comment:
```suggestion
import type { Feature, FeatureCollection, LineString, Point } from "geojson";
```
Bitte einmal ```npm remove geojson``` machen und ```npx expo install @types/geojson``` ausführen. 
Die geojson libary ist fürs parsen gedacht und schon lange nicht mehr geupdatet worden. Du brauchst hier nur die Types.